docs/source/usersguide/depletion.rst

```diff
@@ -370,3 +370,67 @@ The values of the one-group microscopic cross sections passed to
 :class:`openmc.deplete.IndependentOperator` are fixed for the entire depletion
 simulation. This implicit assumption may produce inaccurate results for certain
 scenarios.
+
+Transfer Rates
+==============
+
+Transfer rates define removal or feed of nuclides to or from one or more
+depletable materials. This can be useful to model continuous fuel reprocessing,
+online fission products separation, etc.
+
+Transfers are defined calling the
+:meth:`~openmc.deplete.abc.Integrator.add_transfer_rate()` method, directly from
+one of the Integrator classes::
+
+    ...
+    integrator = openmc.deplete.PredictorIntegrator(op, time_steps, power)
+    integrator.add_transfer_rate(...)
+
+Defining transfer rates
+-----------------------
+
+The :meth:`~openmc.deplete.abc.Integrator.add_transfer_rate()` method requires an instance
+to a :class:`~openmc.Material` or the material id or
+the material name, as the depletable material from which nuclides are processed,
+a list of elements that share the same transfer rate, and a transfer rate value, as the
+rate at which elements are transferred.
+
+.. important::
+
+   Make sure you set the transfer rate value with the right sign.
+   A positive transfer rate assumes removal, while a negative one assumes feed.
```

Comment:
This seems backwards to me on first read, but it makes sense in the context of the depletion equation. It may be prudent to use `.. caution::` here for emphasis instead of `.. important::`